client/js/components/supporter-address-form.es6

```diff
@@ -15,11 +14,12 @@ const request = require('../common/request')
 // pass in some default data for your update payload
 function init(state) {
   state = state || {}
-  state = R.merge({
+  state = {
     submit$: flyd.stream()
   , supporter$: flyd.stream(state.supporter || {})
   , error$: flyd.stream()
-  }, state)
+  , ...state
+  }
```

Comment:
Is this the correct order? Do the properties of state go first or second?

Reply:
looks from the docs like the second should override the first, so this order should work the same, yeah
https://ramdajs.com/0.25.0/docs/#merge